webvtt.go

```diff
@@ -59,11 +59,15 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	var item = &Item{}
 	var blockName string
 	var comments []string
+	var indexContent []int
```

Comment:
Instead of storing index contents in a slice and looping in the slice later, we can use a single variable here : `var index int`